world/map/npc/functions/hug.txt

```diff
@@ -1,9 +1,8 @@
 -|script|nonmagic-hug|32767
 {
-    explode .@name$[0], @args$, "*"; // strip the trailing *
```

Comment:
@wushin the explode is required as when you do `/me hugs foo` the name will be `foo*` so we need to strip the trailing asterisk if any